src/read_dump.cpp

```diff
@@ -276,7 +259,7 @@ bigint ReadDump::seek(bigint nrequest, int exact)
 
   // proc 0 finds the timestep in its first reader
 
-  if (me == 0 || parallel) {
+  if (comm->me == 0 || parallel) {
```

Comment:
What is the motivation for this change for "me" and "nprocs" throughout this file.  I think using local copies of these Comm variables is done in several other files as well, for convenience.  Ditto for changing to domain->triclinic instead of triclinic.